### ISSUE TITLE
files we want to be indexed in the housing server

### ISSUE THREAD

**Issue by keighrim** [clamsproject/aapb-brandeis-datahousing#1]
Currently, in the Brandeis internal data directory of AAPB data contains files of lots of different types. Here's the file extensions that could be interest of ours that I can think. 

Leave comments if you think we have include other types. 

``` bash 
$ find /llc_data/clams/wgbh -type f  -exec basename {} \; | rev | cut -d . -f 1 | rev  | sort -u
_auto-update
auto-update
CodeResources
conf
csv
dll
DS_Store
dylib
exe
icns
ico
inf
Installer
json
lsa
manifest
mov  #video
mp3  #audio
mp4  #video
nib
pdf
plist
png  #image
py
qm
QtCore
QtGui
QtNetwork
QtXml
_registration
registration
_resources
SetFile
SetFileLeopard
sh
srt  #text
swp
tmp
txt  #text
vtt  #text
xlsx
xml
zip
```

**Comment by marcverhagen** [comment]
The current server only recognizes the following extensions:

- [vtt txt] ⟹ text
- [xml] ⟹ markup
- [mp3] ⟹ audio
- [mp4 mov webm mkv] ⟹ video
- [everything else] ⟹ other

I am adding srt, png, wav, pdf, json and maybe some others for now.

In a way it would be nice if we could both search for those types and for extensions, I think I will update the code to allow that.

**Comment by keighrim** [comment]
I'm not sure we want to treat pdf files as `TextDocument` since the anchoring mechanism for complex-layout pdf documents (multi-column, floating tables/examples, or even embedded images/figures) can be very different from regular linear 0-index character offsets, not to mention that many pdf documents (especially within archival context) do not have embedded text and thus have to be gone through OCR/TR just for text (with or without proper reading order understanding).

**Comment by marcverhagen** [comment]
Yes, for sure they should not be text. I have not added them so far and once we have a need for them we would add them under another type, and maybe need to expand the vocabulary as well.
